deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md

```diff
@@ -0,0 +1,124 @@
+---
+applies_to:
+  deployment:
+    self:
+navigation_title: Connect your self-managed cluster
+---
+
+# Connect your self-managed cluster to AutoOps
+
+To use AutoOps with your self-managed cluster, you first need to create an {{ecloud}} account or log in to your existing account. Once you choose to connect AutoOps to your self-managed cluster, an installation wizard will guide you through the steps of installing Elastic Agent to send metrics from your self-managed cluster to AutoOps in {{ecloud}}.  
+
+Complete the steps in the following subsections to connect your cluster to AutoOps. The setup only takes about 10 minutes.
+
+## Prerequisites
+
+Ensure you meet the following requirements before proceeding:
+
+* You have an [Enterprise self-managed license](https://www.elastic.co/subscriptions) or an active self-managed [free trial](https://cloud.elastic.co/registration).
+* Your cluster is on a [supported {{es}} version](https://www.elastic.co/support/eol).
```

Comment:
```suggestion
* It's on a [supported {{es}} version](https://www.elastic.co/support/eol).
* You have an [Enterprise self-managed license](https://www.elastic.co/subscriptions) or an active self-managed [free trial](https://cloud.elastic.co/registration).
```
Just trying to rephrase and/or reorder, so that we avoid any implications about the user as opposed to the system.